quarto/config/routes.yaml

```diff
@@ -26,3 +26,38 @@ game_place:
         idGame: '\d+'
         x: '\d+'
         y: '\d+'
+
+gameapi_new:
+  path: /gameapi
```

Comment:
I would use a `prefix` instead of repeating it everywhere: https://symfony.com/blog/new-in-symfony-4-1-prefix-imported-route-names